index.bs

```diff
@@ -725,71 +687,75 @@ Equality for [=ordered sets=] is equality of its members, but without
 regard to order:
 [=Ordered sets=] |A| and |B| are <dfn for=set>equal</dfn> if both |A| is a
 [=superset=] of |B| and |B| is a [=superset=] of |A|.
-
 </div>
 
 ## Defaults ## {#sanitization-defaults}
 
-Note: The defaults should follow a certain form, which is checked for at the
-      beginning of [=canonicalize a configuration=].
+There are four builtins:
 
-The <dfn>built-in default config</dfn> is as follows:
+* The [=built-in safe default config=],
+* the [=built-in unsafe default config=],
+* the [=built-in safe baseline config=], and
+* the [=navigating URL attributes list=].
+
+The <dfn>built-in safe default config</dfn> is the same as the [=built-in safe baseline config=].
+
+ISSUE(233): Determine if this actually holds.
+
+
+The <dfn>built-in unsafe default config</dfn> is meant to allow anything.
```

Comment:
I suspect not. I left this one, since the next step is to re-work the default handling along the lines we discussed in yesterday's meeting, and I hope I can fix all the builtin/default things in one go thhere.